src/components/molecules/pronunciation/pronunciation.scss

```diff
@@ -2,7 +2,7 @@
 
 .pronunciation {
 	//display: flex;
-	font-family: Noto Sans;
```

Comment:
so, this doesn't have Noto Sans in quotes on purpose, meant to match the the font declaration in nci_common.css.  But maybe let's try it out and see if there is a difference in rendering on react-app-dev.